src/custom_node.hpp

```diff
@@ -46,7 +47,7 @@ class CustomNode : public Node {
         const std::unordered_map<std::string, std::string>& nodeOutputNameAlias = {},
         std::optional<uint32_t> demultiplyCount = std::nullopt,
         std::set<std::string> gatherFromNode = {},
-        void* customNodeLibraryInternalManager = nullptr);
+        std::shared_ptr<CNLIMWrapper> customNodeLibraryInternalManager = std::make_shared<CNLIMWrapper>(nullptr, [](void* ptr) -> int { return 0; }));
```

Comment:
Why can't it be just `nullptr` here?

Reply:
It can be nullptr. Will change that.

Reply:
It seems like I was wrong and it cannot be nullptr. That is because CNLIMWrapper has destructor which calls deinitialize(void*). When we pass second argument as nullptr than in destructor we try to call nullptr, which produces error(found that when unit test failed).

Reply:
What about:
```
std::shared_ptr<CNLIMWrapper> customNodeLibraryInternalManager = nullptr
```

Reply:
This way destructor is never called